src/cljam/vcf/reader.clj

```diff
@@ -93,28 +94,32 @@
 ;; ------------------
 
 (defn- parse-data-line
-  [line header]
-  (let [[fields gt-fields] (->> (cstr/split line #"\t")
+  [line header kws]
+  (let [[fields gt-fields] (->> (.split ^String line "\t")
+                                clojure.lang.LazilyPersistentVector/createOwning
```

Comment:
Thank you for the advice.
I've added comments about performance and link to this PR.